docs/dev-guide/integration/events/http.rst

```diff
@@ -1,6 +1,9 @@
 HTTP Notifier
 =================
 
+.. deprecated:: 2.7
+   This page describes a notification framework that has been deprecated and will go away in Pulp 3.0.
```

Comment:
Let's make sure katello is aware we're not just giving them a new option, but also planning to take this one away. They currently depend on this feature.